builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,28 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+
+	local count = 1
+	for i = 9, 1, -1 do
+		local check_path = gamedetails.path .. DIR_DELIM .."menu" ..
+					DIR_DELIM .. identifier .. "." .. i .. ".png"
+		local file = io.open(check_path,"rb")
```

Comment:
Instead of this check it would be better I think to use the result of `core.get_dir_list` to check whether the file is inside.